libs/io_repl.c

```diff
@@ -66,6 +66,12 @@ void b_getchar(int idx, int *cells) {
       putchar(' ');
       cells[idx] = c;
       return;
+    case 0x7f:
```

Comment:
Would you like to rewrite this using a `define` to make things more readable? Something like:

`#define delete_character 0x7f` 

It could be easier to read and also some other people might not know what `0x7f` stands for just as I did when I first saw it :stuck_out_tongue:. What do you think?